packer-plugin-azure.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: packer-plugin-azure
+  version: 2.1.8
+  epoch: 0
+  copyright:
+    - license: MPL-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/hashicorp/packer-plugin-azure
+      tag: v${{package.version}}
+      expected-commit: e6b551be9d5aae257c8c6711ba58a05d2555318a
+
+  - uses: go/bump
+    with:
+      deps: github.com/hashicorp/go-retryablehttp@v0.7.7
+
+  - uses: go/build
+    with:
+      packages: .
+      output: packer-plugin-azure
+      ldflags: -X github.com/hashicorp/packer-plugin-azure/version.Version=${{package.version}}
+
```

Comment:
Re: testing. In the absence of a --help or a --version, can you share info on how we were able to manually test this? It seems like it requires packer to operate, and I don't believe we have a packer apk today

Reply:
Have we been able to do any manual validation that the plugins are detected and usable by packer?